### PR TITLE
preparing 7.15.0

### PR DESCRIPTION
**Pull request by robertohuertasm**

### CODE REVIEW

**Comment by JimiC** [comment]
Hold your horses. I'm not done.

**Comment by jens1o** [comment]
lol.

**Comment by JimiC** [comment]
@robertohuertasm `JSON5` is the last PR for this sprint.

**Comment by robertohuertasm** [comment]
Merged!

File: README.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ If you're willing to collaborate with us feel free to join our [Github repositor
 
 ## State of the extension
 
+Our main core maintainer, [@JimiC](https://github.com/JimiC), has worked a lot for **7.15.0** and there are a lot of new cool icons in this new version. We hope that you enjoy them!
```
Comment (JimiC) [inline README.md]: Indeed. My skills with `Illustrator` have improved. 😸

**Comment by jens1o** [comment]
what about doing this release as 8.0.0? (Translation files have been moved and the Translation Service will launch?)

**Comment by JimiC** [comment]
Nope. The API hasn't changed. No breaking changes. `7.15.0` is justified.

**Comment by robertohuertasm** [comment]
Awesome work @JimiC! 🥇

**Pull request merged**